indexer/feature_data.cpp

```diff
@@ -341,7 +341,7 @@ bool FeatureParams::AddHouseNumber(string houseNumber)
   ASSERT_NOT_EQUAL(houseNumber.front(), ' ', ("Trim should be done by the caller."));
 
   // Negative house numbers are not supported.
-  if (houseNumber.front() == '-' || houseNumber.find(u8"－") == 0)
+  if (houseNumber.front() == '-' || houseNumber.find(reinterpret_cast<char const *>(u8"－")) == 0)
```

Comment:
I'd prefer to remove u8 at all here and below. reinterpret_cast is not a good solution.